--- vendor/composer/autoload_namespaces.php.orig	2014-08-16 13:27:20.229662161 +0200
+++ vendor/composer/autoload_namespaces.php	2014-08-16 13:30:56.465332230 +0200
@@ -6,12 +6,12 @@
 $baseDir = dirname($vendorDir);
 
 return array(
-    'fkooman\\Rest\\' => array($vendorDir . '/fkooman/rest/src'),
+    'fkooman\\Rest\\' => array('/usr/share/php'),
     'fkooman\\OAuth\\Server\\' => array($baseDir . '/src'),
-    'fkooman\\OAuth\\Common\\' => array($vendorDir . '/fkooman/oauth-common/src'),
-    'fkooman\\Json\\' => array($vendorDir . '/fkooman/json/src'),
-    'fkooman\\Http\\' => array($vendorDir . '/fkooman/rest/src'),
-    'fkooman\\Config\\' => array($vendorDir . '/fkooman/config/src'),
-    'Twig_' => array($vendorDir . '/twig/twig/lib'),
-    'Symfony\\Component\\Yaml\\' => array($vendorDir . '/symfony/yaml'),
+    'fkooman\\OAuth\\Common\\' => array('/usr/share/php'),
+    'fkooman\\Json\\' => array('/usr/share/php'),
+    'fkooman\\Http\\' => array('/usr/share/php'),
+    'fkooman\\Config\\' => array('/usr/share/php'),
+    'Twig_' => array('/usr/share/pear'),
+    'Symfony\\Component\\Yaml\\' => array('/usr/share/php'),
 );
